posthog/api/test/__snapshots__/test_decide.ambr

```diff
@@ -65,7 +65,7 @@
          "posthog_team"."event_properties_numerical"
   FROM "posthog_team"
   WHERE "posthog_team"."id" = 2
-  LIMIT 21 /*controller='team-detail',route='api/projects/%28%3FP%3Cid%3E%5B%5E/.%5D%2B%29/%3F%24'*/
```

Comment:
were we expecting to lose these comments? do we care?

Reply:
They're back!